book/src/configuration.md

```diff
@@ -118,9 +118,11 @@ The following statusline elements can be configured:
 | `enable`              | Enables LSP integration. Setting to false will completely disable language servers regardless of language settings.| `true` |
 | `display-messages`    | Display LSP progress messages below statusline[^1]          | `false` |
 | `auto-signature-help` | Enable automatic popup of signature help (parameter hints)  | `true`  |
+| `display-inlay-hints` | Display inlay hints[^2]                                     | `true`  |
```

Comment:
```suggestion
| `display-inlay-hints` | Display inlay hints[^2]                                     | `false`  |
```

For now I don't think we should enable this by default until the implementation is improved further from this MVP.